modifiers/heading_id.js

```diff
@@ -24,9 +24,16 @@ const safeObject = require('../util/safe_object');
 
 /**
  * @typedef {Object} HeadingIdOptions
+ * @property {HashLinkOptions} [hashLink] Override default hash link options.
  * @property {function(string): string} [sluggify] Custom function to convert heading content to a slug Id.
  */
 
+/**
+ * @typedef {Object} HashLinkOptions
+ * @property {number} maxLevel Max number of heading levels to generate a hash link.
+ * @property {string} class Class name to use on the hash link.
```

Comment:
```suggestion
 * @property {number} [maxLevel=3] Max heading level to generate hash links for.
 * @property {string} [class='hash-anchor'] Class name to use on the hash link.
```
Adding the default values, and the nitpick on wording from the README